Automattic/wp-calypso
# State: Remove unused media selectors and reducers

There is some unused media state (selectors and reducers) and this PR removes it.

It's expected that there will be a small reduction in bundle sizes, because this removes portions of the media reducer that were still being loaded.

#### Changes proposed in this Pull Request

* State: Remove unused media selectors and reducers

#### Testing instructions

* One by one, verify none of the removed items (selectors, reducer trees) is used anywhere.
* Verify all tests pass.
